src/components/shared/Title.component.tsx

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+import type { TitleProps } from '@/interfaces/TitleProps.ts';
+
+export const TitleComponent: React.FC<TitleProps> = ({ children, titleClasses }) => <h2 className={titleClasses}>{children}</h2>;
```

Comment:
Please add a proper `line-hegth`.
Actually, I propose to keep specific styles for this title, as `font-size` and `line-hegth` inside of this component

Reply:
No, I just mean, that all titles in the project have same font size and line height, so if we have separate component for the title, we can keep these common props in it

Reply:
ok, fix in the next pr, please